### ISSUE TITLE
deal with missing trailing zeros

### ISSUE THREAD

**Issue by johnsanterre** [dssg/eights#56]
Some of these db's do csv dumps that miss the final empty entries.  it seems to be only the trailing values.

fast cheap solution is:  do we propose a better one?

def append_trailing_comas(L_L, target=None):
    if target==None:
        target = len(L_L[0])
    for x in range(len(L_L)):
        if len(L_L[x]) != target:
            r = target-len(L_L[x])
            for y in range(r):
                L_L[x].append('')
    return L_L

**Comment by zar1** [comment]
Truly, there is an infinitude of things that can go wrong when reading CSVs. Every day brings new and surprising problems the CSV reading. I am in awe. I think it's harder to read a CSV than it is to implement The Internet.

**Comment by johnsanterre** [comment]
True, given this is required to validate an email, (http://www.ex-parrot.com/pdw/Mail-RFC822-Address.html) how could we expect to validate a csv!
